salt/journal/config/etc-nginx-sites-available-journal.conf

```diff
@@ -87,6 +87,9 @@ server {
         rewrite '^/Events$' '{{ public_scheme }}://{{ public_host }}/events' permanent;
         rewrite '^/Labs$' '{{ public_scheme }}://{{ public_host }}/labs' permanent;
 
+        # rewrite to external URLs have to go here
+        rewrite '^/highlights$' 'https://crm.elifesciences.org/crm/highlights' permanent;
```

Comment:
We can look to introduce a different approach if we find ourselves maintaining many external redirects.

Reply:
Indeed extracting a `map` like for the other file is a mechanical refactoring. The same approach with an `if` could also be used, but `if`s in nginx work in counterintuitive ways so would need careful testing. I do think if anyone wants to know what redirects `/highlights` this place is easy to find, and a developer adding a new redirec can replicate the same structure with a new `rewrite` directive.